src/task_manager.rs

```diff
@@ -47,6 +58,24 @@ impl TaskManager {
         (*self.exchange_rates.lock().unwrap()).clone()
     }
 
+    pub fn get_lsp_fee(&self, exchange_rate: &Option<ExchangeRate>) -> Result<LspFee> {
+        let mut locked_lsp_fee = self.lsp_fee.lock().unwrap();
+        let lsp_fee_cache = match &mut *locked_lsp_fee {
+            Some(l) => Ok(l),
+            None => Err(RuntimeError {
+                code: RuntimeErrorCode::LspServiceUnavailable,
+                msg: "Cached lsp fee isn't available".to_string(),
+            }),
+        }?;
+
+        Ok(LspFee {
+            channel_minimum_fee: lsp_fee_cache
+                .channel_minimum_fee
+                .to_amount_down(exchange_rate),
```

Comment:
We should be pessimistic from the user's POV. As this is a fee, it's better to round up.
```suggestion
                .to_amount_up(exchange_rate),
```